### PR TITLE
Fix "typo" and clearing CSS

### PR DESCRIPTION
**Pull request by iap**
blog.css :

> - Removing comma at font: normal bold 11px 'Roboto',,sans-serif;
> - Removing <style></style>
> 

style_amadeus.css, style_ancestry.css, style_blablacar.css, style_box.css, style_case_studies.css, style_crowdfire.css, style_haufegroup.css, style_huawei.css, style_wink.css, style_zalando.css:

> - Add tab or "space" on @media screen and (max-width: 910px)
> - Add close brackets "}" /* End Media 910px */
> - Change some typo "magin" to "margin". magin-left:10px;

style_user_journeys.css:

> - Fill in number "0%" at padding-top:%;
> - Removing at background-color:#3371e3 !important;
> - Change "colon" to "semicolon" at margin-bottom:10%:
> - Removing "$" at background-color: $light-grey;

### CODE REVIEW

**Comment by bradtopol** [comment]
HI @mistyhacks would you be willing to review this? It is css related.

File: static/css/blog.css [hunk 1]
```diff
@@ -321,8 +321,7 @@ div.post-footer {
   font-size: 16px;
   line-height: 1.5;
 }
---></style>
```
Comment (mistyhacks) [inline static/css/blog.css]: I'm not sure what this was even doing here -- can you blame it and see what commit added it?

File: static/css/blog.css [hunk 2]
```diff
@@ -336,14 +335,16 @@ body {
...
 }
+
 .main-inner .fauxcolumn-center-outer {
   left: 0px;
   right: 310px;
   /* IE6 does not respect left and right together */
   _width: expression(this.parentNode.offsetWidth -
-    parseInt("0px") -
-    parseInt("310px") + 'px');
+  parseInt("0px") -
```
Comment (mistyhacks) [inline static/css/blog.css]: I'm unfamiliar with this syntax at all. Did the indentation break something?

File: static/css/style_ancestry.css [hunk 3]
```diff
@@ -446,4 +447,5 @@ h2 {
     padding-bottom:5%;
     padding-top:2%;
   }
+
```
Comment (mistyhacks) [inline static/css/style_ancestry.css]: Nit: Probably unnecessary whitespace change

File: static/css/style_blablacar.css [hunk 4]
```diff
@@ -255,203 +255,204 @@ h4 {
...
 
-.header_logo {
-  width:35%;
-  margin-bottom:-.5%;
-  magin-left:10px;
-}
+  .header_logo {
+    width:35%;
+    margin-bottom:-.5%;
+    margin-left:10px;
```
Comment (mistyhacks) [inline static/css/style_blablacar.css]: 😂

File: static/css/style_blackrock.css [hunk 5]
```diff
@@ -254,206 +254,207 @@ h4 {
...
-  width:35%;
-}
+  .logo {
+    width:35%;
+  }
+} /* End Media 910px */
 
 @media screen and (max-width: 580px){
-
+
```
Comment (mistyhacks) [inline static/css/style_blackrock.css]: nit: whitespace

File: static/css/style_buffer.css [hunk 6]
```diff
@@ -256,199 +256,200 @@ h4 {
...
+    padding-top:5%;
+    padding-bottom:5%;
+    font-size:18px;
+    letter-spacing:0.03em;
+    line-height:24px;
+    width:80%;
+    padding-left:15%;
+    padding-right:10%;
+    float:left;
+    background: url('/images/CaseStudy_buffer_banner1.jpg');
```
Comment (mistyhacks) [inline static/css/style_buffer.css]: Nice

File: static/css/style_user_journeys.css [hunk 7]
```diff
@@ -245,7 +245,7 @@ border:1px solid blue;
 
 .docscoltitle {
 float:left;
-padding-top:%;
+padding-top:0%;
```
Comment (mistyhacks) [inline static/css/style_user_journeys.css]: I don't think `0%` is valid -- I think it just needs to be `0`.

File: static/css/style_user_journeys.css [hunk 8]
```diff
@@ -413,7 +409,7 @@ button {
 
 .navButton:focus {
 
-  background-color:#3371e3 !important</div>;
+  background-color:#3371e3 !important;
```
Comment (mistyhacks) [inline static/css/style_user_journeys.css]: Nice (weird!)

File: static/css/style_user_journeys.css [hunk 9]
```diff
@@ -897,7 +893,7 @@ table {
 }
 
 thead, tr:nth-child(even) {
-  background-color: $light-grey;
+  background-color: light-grey;
```
Comment (mistyhacks) [inline static/css/style_user_journeys.css]: wow

**Comment by mistyhacks** [comment]
/assign @kbarnard10 

Please look at the case studies and blog and verify these changes don't break anything there.

**Comment by mistyhacks** [comment]
I reviewed the source changes and had some questions -- I am unfamiliar with some of the syntax used here. (That doesn't make it wrong or bad).

I reviewed the smoke test page (https://deploy-preview-10246--kubernetes-io-master-staging.netlify.com/docs/test/) and it looks fine in this PR. I wonder what `!important` things we will be able to drop in our CSS now that some of these missing braces and semicolons will mean that the rules actually get read all the way through.

/lgtm

File: static/css/blog.css (hunk 1 above)
Comment (iap) [reply static/css/blog.css]: https://github.com/iap/website/blame/88a76d7632adec440912ffb31e90de19969929b6/static/css/blog.css

**Comment by Bradamant3** [comment]
@iap with apologies for the delay -- it looks as though this PR is ready to merge, but it needs a rebase on a fresh master. As soon as you rebase, we'll get it merged.

And many thanks for the painstaking work!

**Comment by mistyhacks** [comment]
We'd love to accept this, but it has some pending feedback and needs to be rebased to resolve conflicts. Thanks!

/hold

File: static/css/blog.css (hunk 2 above)
Comment (iap) [reply static/css/blog.css]: Hi, @mistyhacks . The syntax didn't break something, I just tested with the other syntax and revert it back as before.

.main-inner .fauxcolumn-center-outer {
        left: $(main.column.left.width);
        right: $(main.column.right.width);
        /* IE6 does not respect left and right together */
        _width: expression(this.parentNode.offsetWidth -
            parseInt("$(main.column.left.width)") -
            parseInt("$(main.column.right.width)") + 'px');
      }
